index.html

```diff
@@ -81,6 +81,8 @@
         openmct.install(openmct.plugins.example.EventGeneratorPlugin());
         openmct.install(openmct.plugins.example.ExampleImagery());
         openmct.install(openmct.plugins.example.ExampleTags());
+        openmct.install(openmct.plugins.example.ExampleUser());
```

Comment:
not sure if we want to include these, but it helps for testing!

Reply:
If we revert the install from index, the e2e tests will need to be updated to inject the plugins again.